projects/solar/src/mppt.c

```diff
@@ -0,0 +1,42 @@
+#include "gpio.h"
+#include "mux.h"
+#include "spv1020_mppt.h"
+
+// will get the address and size of the demultiplexer
+static MuxAddress s_mux_address = {
+  .bit_width = 3,
+  .sel_pins[0] = { .port = GPIO_PORT_B, .pin = 3 },
+  .sel_pins[1] = { .port = GPIO_PORT_B, .pin = 4 },
+  .sel_pins[2] = { .port = GPIO_PORT_B, .pin = 5 },
+  .mux_enable_pin = { .port = GPIO_PORT_A, .pin = 1 },  // need to check hardward for correct pin
```

Comment:
HW hasn't actually given us an enable pin - we'll need to set this to some pin that isn't being used, like PB1. (The unused pin should be #define'd so we can move it to a config file later and use the same unused pin everywhere we need one.) Also, I believe you'll also have to fill in the `mux_output_pin` parameter with an unused pin. (A better solution of course would be to refactor the `mux` driver so it doesn't need these pins, but alas that's a problem for another ticket.)